crate/operator/grand_central.py

```diff
@@ -161,7 +159,7 @@ def get_grand_central_deployment(
                                     path="/api/health",
                                     port=GRAND_CENTRAL_BACKEND_API_PORT,
                                 ),
-                                initial_delay_seconds=60,
+                                initial_delay_seconds=180,
```

Comment:
Was 60 not enough?

Reply:
Sometimes it takes longer to provision the cluster. So I think it makes sense to wait a bit longer before doing the liveness check..